docs/architecture/anti-entropy/hinted-handoff.rst

```diff
@@ -43,4 +43,65 @@ Hinted handoff is enabled and managed by these settings in :code:`scylla.yaml`:
+
+ScyllaDB should return a JSON containing the ID of the created sync point as the response. In case of an error, a JSON reporting the issue will be returned instead.
+
+Examples:
+
+.. code-block::
+
+    $ curl -X POST http://127.0.0.1:10000/hinted_handoff/sync_point?target_hosts=127.0.0.2,127.0.0.3
+    "Ay4AAADgR8IYRSM14vPuN232DLKSAQAMAAAAAAAAAAAAAAAMAAAAAAAAAAAAAAA/CncIC5VDTA=="
+
+    $ curl -X POST "http://127.0.0.1:10000/hinted_handoff/sync_point"
+    "Ay4AAADgR8IYRSM14vPuN232DLKSAQAMAAAAAAAAAAAAAAAMAAAAAAAAAAAAAAA/CncIC5VDTA=="
+
+    $ curl -X POST "http://127.0.0.1:10000/hinted_handoff/sync_point?target_hosts=invalid_host"
+    {"message": "Failed to parse host address invalid_host: invalid_host", "code": 400}
+
+You can await a previously created sync point by sending a GET request to :code:`/hinted_handoff/sync_point` with the following parameters:
+
+* :code:`id`: the ID of the sync point the user wants to await. This parameter is mandatory. Only one ID can be passed,
+* :code:`timeout`: an integer value specifying how many seconds ScyllaDB should wait for the hints corresponding to the sync point to be sent. If the value is negative, ScyllaDB will wait indefinitely long. This parameter is not mandatory; if it's not present, ScyllaDB will return the result immediately.
+
+The database should return a JSON containing one of the two values as the response:
+
+* :code:`"DONE"` -- if all of the hints corresponding to the passed sync point have already been sent,
+* :code:`"IN_PROGRESS"` -- if some of those hints have not been sent yet.
+
+In case of an error, a JSON reporting the issue will be returned instead.
+
+Examples:
+
+.. code-block::
+
+    $ curl -X GET "http://127.0.0.1:10000/hinted_handoff/sync_point?id=Ay4AAADgR8IYRSM14vPuN232DLKSAQAMAAAAAAAAAAAAAAAMAAAAAAAAAAAAAAA/CncIC5VDTA==&timeout=2"
+    "DONE"
+
+    $ curl -X GET "http://127.0.0.1:10000/hinted_handoff/sync_point?id=Ay4AAADgR8IYRSM14vPuN232DLKSAQAMAAAAAAAAAAAAAAAMAAAAAAAAAAAAAAA/CncIC5VDTA=="
+    "DONE"
+
+    $ curl -X GET "http://127.0.0.1:10000/hinted_handoff/sync_point?timeout=-1"
+    {"message": "Missing mandatory parameter 'id'", "code": 400}
+
+    $ curl -X GET "http://127.0.0.1:10000/hinted_handoff/sync_point?id=invalid_id&timeout=-1"
+    {"message": "Failed to parse the sync point description invalid_id: Base64 encoded length is expected a multiple of 4 bytes but found: 10", "code": 400}
+
+    $ curl -X GET "http://127.0.0.1:10000/hinted_handoff/sync_point?id=invalidIdddd&timeout=-1"
+    {"message": "Failed to parse the sync point description invalidIdddd: Unsupported sync point format version: 138", "code": 400}
+
+
+.. note:: If the node is turned off while you're awaiting a sync point on it, you'll receive an error JSON message as the response. You should try awaiting the sync point again when the node has been turned on again.
```

Comment:
> you'll receive an error JSON message as the response

Not sure if it's necessary to be so specific that you will receive an "error JSON message". You can just say that you will get an error if this happens. JSON is just a format for encoding the error, it's already explained in previous sections.